core/players-handbook-2024/archetypes/wizard-abjurer.xml

```diff
@@ -0,0 +1,114 @@
+
+			<select type="Spell" name="Abjuration Savant (1-7th Level)" supports="$(spellcasting:slots),Abjuration,$(spellcasting:list)" level="13" spellcasting="Wizard" />
+
+			<select type="Spell" name="Abjuration Savant (1-8th Level)" supports="$(spellcasting:slots),Abjuration,$(spellcasting:list)" level="15" spellcasting="Wizard" />
+
+			<select type="Spell" name="Abjuration Savant (1-9th Level)" supports="$(spellcasting:slots),Abjuration,$(spellcasting:list)" level="17" spellcasting="Wizard" />
+
+		</rules>
+	</element>
+
+	<element name="Level 3: Arcane Ward" type="Archetype Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_ARCHETYPE_WIZARD_ABJURER_ARCANE_WARD">
+		<requirements>!ID_INTERNAL_PHB24_FEATURE_REPLACEMENT_WIZARD_ABJURATION_ARCANE_WARD</requirements>
+		<description>
+			<p>You can weave magic around yourself for protection. When you cast an Abjuration spell with a spell slot, you can simultaneously use a strand of the spell's magic to create a magical ward on yourself that lasts until you finish a Long Rest. The ward has a Hit Point maximum equal to twice your Wizard level plus your Intelligence modifier. Whenever you take damage, the ward takes the damage instead, and if you have any Resistances or Vulnerabilities, apply them before reducing the ward's Hit Points. If the damage reduces the ward to 0 Hit Points, you take any remaining damage. While the ward has 0 Hit Points, it can't absorb damage, but its magic remains.</p>
+			<p class="indent">Whenever you cast an Abjuration spell with a spell slot, the ward regains a number of Hit Points equal to twice the level of the spell slot. Alternatively, as a Bonus Action, you can expend a spell slot, and the ward regains a number of Hit Points equal to twice the level of the spell slot expended.</p>
+			<p class="indent">Once you create the ward, you can't create it again until you finish a Long Rest.</p>
+		</description>
+		<sheet alt="Arcane Ward" usage="1/Long Rest">
+			<description>When you cast an Abjuration spell with a spell slot, you can simultaneously use a strand of the spell's magic to create a magical ward on yourself that lasts until you finish a Long Rest. The ward has a Hit Point maximum of {{arcane ward:hp}}. Whenever you take damage, the ward takes the damage instead, and if you have any Resistances or Vulnerabilities, apply them before reducing the ward's Hit Points. If the damage reduces the ward to 0 Hit Points, you take any remaining damage. While the ward has 0 Hit Points, it can't absorb damage, but its magic remains. Whenever you cast an Abjuration spell with a spell slot, the ward regains a number of Hit Points equal to twice the level of the spell slot. Alternatively, as a Bonus Action, you can expend a spell slot, and the ward regains a number of Hit Points equal to twice the level of the spell slot expended.</description>
```

Comment:
```suggestion
			<description>When you cast an Abjuration spell with a spell slot, you can create a magical ward on yourself that lasts until you finish a Long Rest. The ward has a HP maximum of {{arcane ward:hp}}. When you take damage, apply any of your relevant Resistances or Vulnerabilities, the ward then takes the damage instead. If the damage reduces the ward to 0 HPs, you take any remaining damage. While the ward has 0 HPs, it can't absorb damage, but it remains. Whenever you cast an Abjuration spell with a spell slot, the ward regains a number of HPs equal to twice the spell slot's level. Alternatively, as a Bonus Action, you can expend a spell slot, and the ward regains a number of HPs equal to twice the level of the spell slot expended.</description>
```